bpf/headers/go_types.h

```diff
@@ -0,0 +1,56 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I think this file and probe.bpf.c should be combined into one single header file, named appropriately to signify they are related to the extraction of the traceparent, while preserving the copyright and the license of the code.

Reply:
OK, combined file is go_traceparent.h.